src/environment.py

```diff
@@ -248,7 +248,9 @@ def generate(config: dict, app_name: str, site_url: str, db_uri: str) -> dict:
             {
                 "MM_FILESETTINGS_DRIVERNAME": "amazons3",
                 "MM_FILESETTINGS_MAXFILESIZE": config["max_file_size"] * 1048576,  # LP:1881227
-                "MM_FILESETTINGS_AMAZONS3SSL": "true",  # defaults to true; belt and braces
+                "MM_FILESETTINGS_AMAZONS3SSL": "true"
+                if config.get("s3_tls", True)
+                else "false",  # defaults to true; belt and braces
```

Comment:
What is meant by `belt and braces` please?

Reply:
I actually don't know :)  
This comment "belt and braces" was already there before

Reply:
Maybe we can check who added it and figure out what it means or remove it?

Reply:
I removed it